packages/cardano-services/src/NetworkInfo/DbSyncNetworkInfoProvider/DbSyncNetworkInfoProvider.ts

```diff
@@ -14,52 +14,55 @@ import { NetworkInfoCacheKey } from '.';
 import { Pool } from 'pg';
 import { Shutdown } from '@cardano-sdk/util';
 import { loadGenesisData, toGenesisParams, toLedgerTip, toNetworkInfo, toWalletProtocolParams } from './mappers';
-import { pollDbSync } from './utils';
+import { pollService } from './utils';
 
 export interface NetworkInfoProviderProps {
   cardanoNodeConfigPath: string;
-  dbPollInterval: number;
+  pollInterval: number;
 }
 export interface NetworkInfoProviderDependencies {
   db: Pool;
   cache: InMemoryCache;
   logger?: Logger;
+  timeSettingsProvider: TimeSettingsProvider;
 }
 export class DbSyncNetworkInfoProvider extends DbSyncProvider implements NetworkInfoProvider {
   #logger: Logger;
   #cache: InMemoryCache;
   #builder: NetworkInfoBuilder;
   #genesisDataReady: Promise<GenesisData>;
-  #dbPollInterval: number;
+  #pollInterval: number;
   #pollService: Shutdown | null;
+  #timeSettingsProvider: TimeSettingsProvider;
 
   constructor(
-    { cardanoNodeConfigPath, dbPollInterval }: NetworkInfoProviderProps,
-    { db, cache, logger = dummyLogger }: NetworkInfoProviderDependencies
+    { cardanoNodeConfigPath, pollInterval }: NetworkInfoProviderProps,
+    { db, cache, logger = dummyLogger, timeSettingsProvider }: NetworkInfoProviderDependencies
```

Comment:
```suggestion
    { db, cache, logger = dummyLogger, stateQuery }: NetworkInfoProviderDependencies
```

Reply:
Done in 563e7b30ca19d1de3e518e4fc451137b91d658ca